src/test/java/LogoutTest.java

```diff
@@ -0,0 +1,15 @@
+import laba.pages.LoginPage;
+import laba.pages.ProductsPage;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LogoutTest extends BaseTest {
+    @Test
+    public void testLogout() {
+        ProductsPage productsPage = loginAsStandardUser();
+        Assert.assertTrue(productsPage.isPageOpened(), "Products page is opened after login.");
+        LoginPage loginPage = productsPage.logout();
```

Comment:
I don't see that you use header component here productsPageBase.getHeaderMenu().clickHamburgerIcon();